packages/renderer/src/lib/cronjob/CronJobColumnName.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+import { router } from 'tinro';
+
+import type { CronJobUI } from './CronJobUI';
+
+let {
+  object,
+}: {
+  object: CronJobUI;
+} = $props();
```

Comment:
we use a Props interface for defining properties

https://svelte.dev/docs/svelte/$props#Type-safety


```ts
<script lang="ts">
	interface Props {
		adjective: string;
	}

	let { adjective }: Props = $props();
</script>
```